MojioSDK/mojio-sdk-model/src/main/java/io/moj/mobile/android/sdk/model/enums/DistanceUnit.java

```diff
@@ -0,0 +1,47 @@
+package io.moj.mobile.android.sdk.model.enums;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Enum for DistanceUnit types.
+ * Created by mhorie on 2016-01-14.
+ */
+public enum DistanceUnit {
+
+    @SerializedName("Kilometers")
+    KILOMETERS("Kilometers"),
+
+    @SerializedName("Miles")
+    MILES("Miles"),
+
+    @SerializedName("NauticalMiles")
+    NAUTICAL_MILES("NauticalMiles"),
+
+    @SerializedName("Meters")
+    METERS("Meters"),
+
+    @SerializedName("GUnCentiMeterits")
```

Comment:
SerializedName here doesn't match the key (GUnCentiMeterits vs CentiMeter), is this intentional?